content/en/blog/_posts/2024-08-19-pod-failure-policy-for-jobs-goes-ga.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Pod failure policy for Jobs goes GA"
+date: 2024-08-19
+slug: kubernetes-1-31-pod-failure-policy-for-jobs-goes-ga
+author: >
+  Michał Woźniak (Google)
+  Shannon Kularathna (Google)
```

Comment:
```suggestion
  Michał Woźniak (Google), Shannon Kularathna (Google)
```

Reply:
From what I saw in all other blogs authors are listed per line in code: https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2024-04-26-structured-authz-beta.md?plain=1#L6-L11, but adding a GH handle might be nice. I will do that.

Reply:
done